x-pack/test/security_solution_api_integration/test_suites/investigation/saved_objects/tests/notes.ts

```diff
@@ -400,21 +401,39 @@ export default function ({ getService }: FtrProviderContextWithSpaces) {
         });
       });
 
-      it('should return nothing if no notes have been created by that user', async () => {
+      // TODO we need to figure out how to create another user in the test environment
```

Comment:
It might be worth to add a test for `deleted` user as @michaelolo24 also pointed out.

Reply:
how would a deleted user be different from a wrong user id?

Reply:
Yes, i guess may be it will be same. Please ignore my comment.